### ISSUE TITLE
Content editor icon not loaded correctly

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco-CMS#11281]
### Which *exact* Umbraco version are you using? For example: 8.13.1 - don't just write v8

8.17.0

### Bug summary

After pulling latest changes from `v8/contrib` branch, I noticed existing selected icons in header was not loaded correctly and fallback to default `icon-document`.

![image](https://user-images.githubusercontent.com/2919859/136056765-bd0eac94-3e97-47c0-8de2-7f77135577e7.png)

Even when re-selecting icon and saving the document type again it set `icon-document`.

### Steps to reproduce

Select a custom icon on a document type and note is showing `icon-document` when refreshing page.

**Comment by bjarnef** [comment]
@nul800sebastiaan do you know if something has recently changed regarding this? 🤔

**Comment by nul800sebastiaan** [comment]
Maybe this one? https://github.com/umbraco/Umbraco-CMS/pull/11008

**Comment by bjarnef** [comment]
Yes, it add `icon` to query params and if null it fallback to `icon-document`.
But that won't work for all existing document types, which by default doesn't have `icon` querystring param.

In this example route only has the following: `/umbraco#/settings/documentTypes/edit/1150?mculture=en-US`

**Comment by bjarnef** [comment]
Fixed in https://github.com/umbraco/Umbraco-CMS/pull/11283

**Comment by bjarnef** [comment]
Another strange issue I noticed is when creating an element type it also creates a template 🤔
